### PR TITLE
docs(docs/features/software-templates/writing-custom-field-extensions.md): update doc with valid example.

### PR DESCRIPTION
**Pull request by padupe**
## Hey, I just made a Pull Request!

Documentation update after encountering an error following the current one.
As per [issue](https://github.com/backstage/backstage/issues/15450), resolution for use was indicated.
I thought it was worth sharing with the community.

#### :heavy_check_mark: Checklist
- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/features/software-templates/writing-custom-field-extensions.md [hunk 1]
```diff
@@ -23,18 +23,19 @@ the `Scaffolder` frontend plugin in your own `App.tsx`.
...
 You can create your own Field Extension by using the
 [`createScaffolderFieldExtension`](https://backstage.io/docs/reference/plugin-scaffolder.createscaffolderfieldextension)
-`API` like below:
+`API` like below.
+
+As an example, we will create a component that validates whether a string is in the "Kehab case" pattern:
 
 ```tsx
-//packages/app/src/scaffolder/MyCustomExtension/MyCustomExtension.tsx
+//packages/app/src/scaffolder/ValidateKehabCase/ValidateKehabCaseExtension.tsx
```
Comment (benjdlambert) [inline docs/features/software-templates/writing-custom-field-extensions.md:31]: So I think we need to rename `Kehab` to `Kebab` in all the locations here :)

File: docs/features/software-templates/writing-custom-field-extensions.md [hunk 2]
```diff
@@ -23,18 +23,19 @@ the `Scaffolder` frontend plugin in your own `App.tsx`.
 
 You can create your own Field Extension by using the
 [`createScaffolderFieldExtension`](https://backstage.io/docs/reference/plugin-scaffolder.createscaffolderfieldextension)
-`API` like below:
+`API` like below.
+
+As an example, we will create a component that validates whether a string is in the "Kehab case" pattern:
```
Comment (benjdlambert) [inline docs/features/software-templates/writing-custom-field-extensions.md:28]:
```suggestion
As an example, we will create a component that validates whether a string is in the `kebab-case` pattern:
```

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Hey thanks for this! Just a few things on the naming that we should probably address first :)

**Comment by padupe** [comment]
@benjdlambert Thanks!
I made the indicated adjustments.
Thank you for your attention during the review.

**Comment by benjdlambert** [comment]
@padupe it looks like there's still a lot of `kehab` when it should be `kebab`. If you do a find and replace that should fix it :)

**Comment by padupe** [comment]
Really!
Correction made.

**Pull request merged**